sqs/src/main/scala/uk/gov/nationalarchives/DASQSClient.scala

```diff
@@ -52,7 +52,7 @@ class DASQSClient[F[_]: Async](sqsAsyncClient: SqsAsyncClient) {
     Async[F].fromCompletableFuture(Async[F].pure(sqsAsyncClient.sendMessage(messageRequest)))
   }
 
-  /** Receives messages from the specified queue. The messages are serialised using the implicit decoder
+  /** Receives messages from the specified queue. The messages are deserialised using the implicit decoder
```

Comment:
I'm unable to write a comment on line 34 & 39, but those should be "serialises/serialised" too, right?

Reply:
Good spot, I've changed them.